README.md

```diff
@@ -5,20 +5,68 @@ NOTE: 'cron-master' library expected and required as a co-dependency
 
 ## Running the Module
 
-### Execution
+### loadJobs
+Grabs the Cron Job config files in the specified directory, initiates and
+starts them:
 
 ```js
-const cronBuilder = require('tke-cron-master-builder')(cronConfigDir, collection, logCleanUpOpts);
+const path = require('path');
+
+const cleanUpOpts = {
+  frequency: '* * 0 * * *', // midnight
+  retentionPeriod: '5 days' // items more than five days old are removed
+};
+
+const jobsDir = path.join(process.cwd(), './cron-jobs');
+
+const cronBuilder = require('tke-cron-master-builder')(
+  jobsDir,
+  'cron-logs', // a mongodb collection name
+  cleanUpOpts
+);
 
 // load all jobs in cronConfigDir
 cronBuilder.loadJobs();
-
-// get handlers for a job we need to start programmatically
-cron.getHandlers(someJobInstance);
 ```
 
-Grabs the Cron Job config files in the specified directory, initiates and
-starts them.
+### getHandlers
+Perhaps you want to attach handlers to a job without loading it via `loadJobs`,
+this will allow you:
+
+```js
+const CM = require('cron-master');
+const cronBuilder = require('tke-cron-master-builder')(
+  jobsDir,
+  'cron-logs',
+  cleanUpOpts
+);
+
+const job = new CM.CronMasterJob({
+  // will trigger the TIME_WARNING event if job runs more than 2 minutes
+  timeThreshold: 2 * 60 * 1000,
+
+  meta: {
+    name: 'my-cron-job'
+  },
+
+  // these are passed to the node-cron module used internally by CronMasterJob
+  cronParams: {
+    cronTime: '* * * * *',
+    onTick: (job, done) => {
+      console.log(new Date(), 'tick happened for our cron job!');
+      done();
+    }
+  }
+});
+
+const handlers = cronBuilder.getEventHandlers(job);
```

Comment:
Oh I see what this is for now... The job you pass to the getEventHandlers function has nothing to do with the cron-builder jobs. Arguably it could be affected by the cleanup job, but only if you also call loadJobs() which starts the cleanup job. (Should the cleanup job be started immediately then?) I guess it's not worth worrying about too much, since it's an internal module...

One suggestion though: maybe instead of getting handlers which have to be applied manually on the job, maybe just have an `applyHandlers(job)` function that does that for you?